04_numeronym/jimbotech/numeronym.go

```diff
@@ -0,0 +1,72 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"strings"
+	"unicode"
+)
+
+var out io.Writer = os.Stdout
+
+// following rules from https://www.revolvy.com/page/Numeronym
+// Program need not convert words with length 3, and should not convert shorter words.
+// If an l (lowercase ell) would occur before a 1 (one), it should be made uppercase.
+// If an I (uppercase eye) would occur before a 1 (one), it should be made lowercase.
+// Input will be printable ASCII with spaces.
```

Comment:
Your known conversions has one non-ASCII rune. You also do not check for printable runes. This sentence is just wrong and should be removed.

Reply:
Done